packages/grafana-ui/src/types/data.ts

```diff
@@ -41,6 +47,10 @@ export interface Field {
   dateFormat?: string; // Source data format
 }
 
+export interface IndexedField extends Field {
+  index: number;
+}
+
```

Comment:
Can we move `IndexedField` to `processSeriesData.ts`?  I'm hoping `data.ts` just represents the core data exchange formats and the rest can be more efemeral/helpers

Reply:
Makes sense :+1: Will fix